docs/src/demo/Sync.tsx

```diff
@@ -181,6 +185,86 @@ export default {
 }
 </script>`;
 
+  const svelteSourceCode = `<script>
+  import Flicking, { FlickingPanel } from "@egjs/svelte-flicking";
+  import "@egjs/svelte-flicking/dist/flicking.css";
+
+  let flicking0;
+  let flicking1;
+  let flicking2;
+
+  function update(flicking, progress) {
+    flicking.camera.lookAt(flicking.camera.range.min + flicking.camera.rangeDiff * progress);
+  };
+</script>
+
+<Flicking bind:this={flicking0} class="mb-4"
+  options={{ align: "prev", bound: true, bounce: 30 }}
+  on:move={e => {
```

Comment:
svelte CustomEvent should be accessed as detail.

https://svelte.dev/tutorial/component-events

Reply:
Uhh, what do you mean?
If you're telling me to add some tutorials about how to use events in svelte, that's described in the other document.

Reply:
The event we use is closer to ({ detail: e }) => {}.

Reply:
Oh, I got the point. I'll fix that 👍